frontend/src/OneBuilding/AcademyDetail.jsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import {
+  View, Text, Image, StyleSheet,
+} from 'react-native';
+
+import attackIcon from '../../assets/troop/attack.png';
+import defenceIcon from '../../assets/troop/defence.png';
+import cookieIcon from '../../assets/troop/cookie.png';
+import AcademyButtons from './AcademyButtons';
+
+const styles = StyleSheet.create({
+  container: {
+    // borderWidth: 1,
+  },
+  row: {
+    flexDirection: 'row',
+    alignItems: 'center',
+  },
+  iconStyle: {
+    width: 16,
+    height: 16,
+  },
+  textStyle: {
+    lineHeight: 24,
+  },
+});
```

Comment:
this is repeating in every building detail file, please reduce redundant code! we have a common styles file for these kind of stuff